logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc

```diff
@@ -0,0 +1,536 @@
+
+
+// [id="elasticsearch-write-requests-rejection-jumps"]
+// == Elasticsearch Write Requests Rejection Jumps
+//
+// .Troubleshooting
+// TBD
+// Note for writer: This is a warning alert and we haven't documented troubleshooting steps for warning alerts yet. I guess you can skip this in currrent release.
+
+[id="elasticsearch-node-disk-low-watermark-reached"]
+== Elasticsearch Node Disk Low Watermark Reached
+
+Elasticsearch does not allocate shards to nodes that https://www.elastic.co/guide/en/elasticsearch/reference/6.8/disk-allocator.html[reach the low watermark].
+
+.Troubleshooting
+
+. Identify the node on which Elasticsearch is deployed.
++
+[source,terminal]
+----
+oc -n openshift-logging get po -o wide
+----
+
+. Check if there are `unassigned shards`.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/health?pretty | grep unassigned_shards
+----
+
+. If there are unassigned shards, check the disk space on each node.
++
+[source,terminal]
+----
+for pod in `oc -n openshift-logging get po -l component=elasticsearch -o jsonpath='{.items[*].metadata.name}'`; do echo $pod; oc -n openshift-logging exec -c elasticsearch $pod -- df -h /elasticsearch/persistent; done
+----
+
+. Check the `nodes.node_name.fs` field to determine the free disk space on that node.
++
+If the used disk percentage is above 85%, the node has exceeded the low watermark, and shards can no longer be allocated to this node.
+
+. Try to increase the disk space on all nodes.
+
+. If increasing the disk space is not possible, try adding a new data node to the cluster.
+
+. If adding a new data node is problematic, decrease the total cluster redundancy policy.
+
+.. Check the current `redundancyPolicy`.
++
+[source,terminal]
+----
+oc -n openshift-logging get es elasticsearch -o jsonpath='{.spec.redundancyPolicy}'
+----
++
+NOTE: If you are using a `ClusterLogging` CR, enter:
++
+[source,terminal]
+----
+oc -n openshift-logging get cl -o jsonpath='{.items[*].spec.logStore.elasticsearch.redundancyPolicy}'
+----
+
+.. If the cluster `redundancyPolicy` is higher than `SingleRedundancy`, set it to `SingleRedundancy` and save this change.
+
+. If the preceding steps do not fix the issue, delete the old indices.
+
+.. Check the status of all indices on Elasticsearch.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- indices
+----
+
+.. Identify an old index that can be deleted.
+
+.. Delete the index.
++
+[source,terminal]
+----
+oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name> -X DELETE
+----
+
+
+.Additional resources
+
+* Search for "redundancyPolicy" in the "Sample `ClusterLogging` custom resource (CR)" in xref:../../logging/config/cluster-logging-configuring-cr.adoc#cluster-logging-configuring-crd_cluster-logging-configuring-cr[About the Cluster Logging custom resource]
```

Comment:
Same issue as above. Your link works, but it takes me to a page which gives me a dead link. Again, I don't see how valuable of a resource this page is. When I go to a page like this one, with pretty much only one link, my reaction is to click it. I think most users would do the same; hence, most users will find a 404 error by going to this "additional resource." 

<img width="1293" alt="Screen Shot 2021-06-17 at 1 37 26 PM" src="https://user-images.githubusercontent.com/70717303/122446897-2a211f80-cf71-11eb-8445-444ee74c26e6.png">